setup.py

```diff
@@ -216,12 +223,14 @@ def members_appended(*ds):
 
     if cythonize and cythonizing:
         cython_opts = dict(
-            # compile .pyx extensions to .c in parallel
-            nthreads=cpu_threads + 1,
             # default language_level will be '3str' starting from Cython 3.0.0,
             # but old cython versions (< 0.29) do not know that, thus we use 3 for now.
             compiler_directives={'language_level': 3},
         )
+        if os.name != 'nt':
+            # compile .pyx extensions to .c in parallel, does not work on windows
+            cython_opts['nthreads'] = cpu_threads + 1
+
         cythonize([posix_ext, linux_ext, freebsd_ext, darwin_ext], **cython_opts)
```

Comment:
is windows_ext missing in the list here?

Reply:
Huh, strange... Why was it working even if the windows_ext was missing in this list?

Reply:
Ah, I see it: `cythonize` is called twice. In the first call `windows_ext` is missing and in the following line `ext_modules` is used which includes `windows_ext`.